spec/models/rules_of_origin/heading_mappings_spec.rb

```diff
@@ -61,13 +61,13 @@
     context 'with unknown scheme code' do
       let(:scheme_code) { 'unknown' }
 
-      it { is_expected.to be_empty }
+      it { is_expected.to eq({}) }
```

Comment:
:+1:

Reply:
Found I was returning an empty array - when results should be a hash - realised the weakness of checking for `empty?` but not type